sources/github/api.js

```diff
@@ -85,3 +85,22 @@ module.exports.getPRsCommits = ({ prs, start, end }) => Promise.all(prs.map(
       || isClosed(pr) && (closed >= updated) // PR closed and not updated after closing
   }).map((r) => ({ ...r, pull_request_url: pr.pull_request_url, pr_html_url: `${pr.html_url}/commits/${r.sha}` }))),
 )).then((data) => data.flat())
+
+module.exports.getReleases = async ({ repos, start, end }) => {
+  const urls = repos.reduce((acc, curr) => {
+    if (acc.indexOf(curr.url) < 0) {
+      acc.push(curr.url)
+    }
+    return acc
+  }, [])
```

Comment:
what about something like `[... new Set(repos.map(({url}) => url))]`?

Reply:
good call, this was pretty much copy/pasta from similar functions within the repo (by copilot lol)